Quality/MK8_1080p/rules.txt

```diff
@@ -18,6 +18,22 @@ formatsExcluded = 0x41A # exclude obvious textures
 overwriteWidth = 960
 overwriteHeight = 540
 
+# Multiplayer-Mode Resolution (2p)
+[TextureRedefine]
+width = 640
+height = 720
+formatsExcluded = 0x41A # exclude obvious textures
+overwriteWidth = 960
+overwriteHeight = 1080
+
+# Multiplayer-Mode Resolution (4p)
+[TextureRedefine]
+width = 640
+height = 360
+formatsExcluded = 0x41A # exclude obvious textures
+overwriteWidth = 960
+overwriteHeight = 1080
```

Comment:
I didn't notice black bars or anything of the sort after extensively testing 1440p both 4K 2P and 4P multiplayer tbh, though I probably should of included screenshots as well. (will add a few soon) 
As far as half-width and normal height for 4P multiplayer it seems thats what Nintendo was already doing for 720p in perfstudio so it only seemed logical to do the same half width-normal res for other target resolutions. 
I didn't notice the conflict specifically for 4P MP and half-res alpha though, I'll look into possible solutions.